shared/ghutil/pullrequest.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// pullrequest.go provides generic functions related to PullRequest
+
+package ghutil
+
+import (
+	"fmt"
+
+	"github.com/google/go-github/github"
+)
+
+const (
+	// PullRequestOpenState is the state of open PullRequest
+	PullRequestOpenState PullRequestState = "open"
```

Comment:
[nit] can we rename this to open, close and all? the type indicates that it is a PRState.

I like concise var names :D

Reply:
since this is cross package var, I would prefer it keep prefix and postfix to avoid potential naming clash